public/js/track.js

```diff
@@ -261,10 +274,28 @@ function addPlaylistEventListeners() {
                         $('#playlistModal').modal('hide');
                         window.open(url, "_blank");
                     })
-                    .catch((errMessage) => {
-                        alert(errMessage);
+                    .catch((error) => {
+
+                        $('#errorAlert').append('<a class="close alert-close">&times</a>' +
+                            '<span>'+'Sorry, we encountered an internal server error. ' + '</span>');
+
+                        if(error.response) {
+                            $('#errorAlert').append('<span>' + error.response.data.errorMsg +'</span>');
+
+                            if(error.response.data.redirectLink) {
+                                setTimeout(function() {
+                                    location.href= error.response.data.redirectLink
+                                }, 5000);
+                            }
```

Comment:
Oh wait I thought we weren't doing the redirecting timeout thing anymore

Reply:
I was thinking on errors, all we are responsible for is showing the alert and providing the error message

Reply:
I think redirecting is nice in the cases I've specified though so users won't feel confused on what action to take next

Reply:
Aight now that I think about it some more it's just errors at the end of the day. Let's go with the redirect link approach